src/olympia/addons/models.py

```diff
@@ -2385,3 +2386,24 @@ class Meta:
     def save(self, *args, **kwargs):
         self.hashed_guid = hashlib.sha256(self.guid.encode()).hexdigest()
         super().save(*args, **kwargs)
+
+
+class AddonBrowserMapping(ModelBase):
+    """
+    Mapping between a Firefox add-on on AMO and an extension ID for a different
+    browser.
```

Comment:
Nit: It's actually the other way around, right ? A mapping between an extension ID for a different browser to a Firefox Add-on on AMO